game/scripts/vscripts/modifiers/player/modifier_player_Asprose.lua

```diff
@@ -0,0 +1,35 @@
+modifier_player_Asprose = class({})
```

Comment:
```suggestion
modifier_player_asprose = class({})
```
替换成小写字母，整个文件都是 可以用vscode搜索替换
ctrl+f 出来
![image](https://user-images.githubusercontent.com/3445793/189900223-ad8e418b-08e3-444a-8e82-6fa15114d3e0.png)
下拉展开
![image](https://user-images.githubusercontent.com/3445793/189900371-69d926b8-26e9-4a9c-ab44-ba5b644f629b.png)
一键替换

Reply:
这里要勾上 区分大小写
![image](https://user-images.githubusercontent.com/3445793/189901415-112147dc-8d67-425a-b2b6-bf0a6a1909ff.png)